specification/migrate/resource-manager/Microsoft.Migrate/2017-11-23-preview/swagger/swagger.json

```diff
@@ -0,0 +1,2780 @@
+{
```

Comment:
Please rename this file to `migrate.json` and don't place it in a folder called `swagger` (ie, should be `.../resource-manager/Microsoft.Migrate/2017-11-23-preview/migrate.json`)